<ui version="4.0" >
 <class>DlgTrainerSelect</class>
 <widget class="QDialog" name="DlgTrainerSelect" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Select trainer</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Trainer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboxTrainer" />
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Lessons to include:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="lwLessons" />
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QCheckBox" name="chkBoxConjs" >
           <property name="text" >
            <string>Include conjugations of verbs</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkBoxDecls" >
           <property name="text" >
            <string>Include declinations of nouns etc.</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>161</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgTrainerSelect</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>259</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel" >
     <x>259</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
